src/towncrier/test/test_build.py

```diff
@@ -1248,6 +1248,49 @@ def test_default_start_string_markdown(self, runner):
 
         self.assertEqual(expected_output, output)
 
+    @with_project(
+        config="""
+        [tool.towncrier]
+        directory = "changes"
```

Comment:
Maybe have explcicit empty configuration ?

```suggestion
        name = ""
        directory = "changes"
```

Reply:
Why that? It doesn't seem to be relevant for this test